packages/parmap/parmap.1.0-rc11/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "Roberto Di Cosmo <roberto@dicosmo.org>"
+authors: "Roberto Di Cosmo <roberto@dicosmo.org>"
+homepage: "https://github.com/rdicosmo/parmap"
+dev-repo: "git+https://github.com/rdicosmo/parmap.git"
+bug-reports: "https://github.com/rdicosmo/parmap/issues"
+build: [
+  ["./configure"] 
+  [make "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib" ]
+]
+install: [
+  [make "install" "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib"]
+]
+remove: [
+  ["./configure"] 
+  [make "uninstall" "DESTDIR=%{prefix}%" "OCAMLLIBDIR=lib"]
+]
+depends: [
+  "ocaml" {< "4.08.0"}
```

Comment:
```suggestion
  "ocaml" {< "4.08.0"}
```
If the big change of this version is `should compile even with OCaml 4.09.0`, why is the constraint on ocaml still the same?

Reply:
I removed the constraint